Client/Frontend/Widgets/AutocompleteTextField.swift

```diff
@@ -150,7 +182,6 @@ class AutocompleteTextField: UITextField, UITextFieldDelegate {
         removeCompletion()
         super.deleteBackward()
         enteredTextLength = count(self.text)
-        autocompleteDelegate?.autocompleteTextField(self, didEnterText: self.text)
```

Comment:
I'm not sure we want to remove this line -- this is what triggers the search results below the text field. That is, if I type "y", I should see all my history entries below that contain "y". After [bug 1166781](https://bugzilla.mozilla.org/show_bug.cgi?id=1166781) this line should be safe to keep hopefully?